### ISSUE TITLE
Auido band settings doesn't show

### ISSUE THREAD

**Issue by RealSimpleMatt** [dsafa/audio-band#274]
https://user-images.githubusercontent.com/43842727/114339050-75e9ce00-9ba8-11eb-89b4-5de7a8db8525.mp4

![Capture](https://user-images.githubusercontent.com/43842727/114339104-9023ac00-9ba8-11eb-9a01-1945e2f6a95d.JPG)

I have net framework 4.8 installed and running on windows 10 20H2

**Comment by eltoncezar** [comment]
First you need to enable the taskbar widget, in Toolbars > Audio Band. Then you right click it, the settings should appears in a popup menu as AudioBand Settings

**Comment by RealSimpleMatt** [comment]
> First you need to enable the taskbar widget, in Toolbars > Audio Band. Then you right click it, the settings should appear in a popup menu as AudioBand Settings.

Im stupid, thank you!

**Comment by otmanzy99** [comment]
Does this work with the Windows 11 taskbar? I am only getting Task Bar Settings and Task Manager when I right click my taskbar.

**Comment by redglitch2** [comment]
> 

yes, but you need to patch the taskbar. You can use [StartAllBack](https://www.startallback.com/) or [ExplorerPatcher](https://github.com/valinet/ExplorerPatcher). Start all back is not free[30 day trial] and has more features, but ExplorerPatcher is free. I would like to note, that ExplorerPatcher did not work on my windows 11 machine, but StartAllBack worked flawlessly.

**Comment by BBBensn** [comment]
A bit late to the party, but just tried to install and activate audio band and got the same problem. I tried to activate it in the windows settings, but even there audio band doesn´t show up. Audioband only shows up in my installed programs windows settings, otherwise its completely ignored by my PC. I tried to reinstall, to repair it, rebooted my pc a few times but it just won´t show up.
Explorerpatch is working fine as far as i see it. 
Any Ideas on how to fix this?

![image](https://github.com/user-attachments/assets/338f85e8-e7e7-44f8-9837-192646adeb59)
![image](https://github.com/user-attachments/assets/dc7ada2a-497c-4c86-a3ea-9215d1a861f4)
![image](https://github.com/user-attachments/assets/0765a4e2-fdfe-46bb-9db3-4b062c81b959)

(everything in German but you´ll get it, second pic just to be sure im in the right settings, theres lots of useless programs in this list but audioband doesn´t show; 3rd pic just to show AudioBand is installed and recognized by windows)
